make fails due to existing symlink

I'm working on tests and changing code in packages in tests/* causes the following error:
```
$ make check GO_TEST_FUNC_ARGS='-run TestPrivateNetCustom'
...
echo -n dir2 >/home/junky/GO/PATH/src/github.com/coreos/rkt/build-rkt-0.7.0+git/tmp/functional-tests/image/rootfs/dir2/file
ln -s /inspect /home/junky/GO/PATH/src/github.com/coreos/rkt/build-rkt-0.7.0+git/tmp/functional-tests/image/rootfs/inspect-link
ln: failed to create symbolic link ‘/home/junky/GO/PATH/src/github.com/coreos/rkt/build-rkt-0.7.0+git/tmp/functional-tests/image/rootfs/inspect-link’: File exists
tests/functional.mk:43: recipe for target '/home/junky/GO/PATH/src/github.com/coreos/rkt/build-rkt-0.7.0+git/tmp/functional-tests/rkt-inspect.aci' failed
make: *** [/home/junky/GO/PATH/src/github.com/coreos/rkt/build-rkt-0.7.0+git/tmp/functional-tests/rkt-inspect.aci] Error 1
```

The line was introduced here https://github.com/coreos/rkt/commit/07c5d35d267bf08a03b8d07273dc9e9303334979#diff-eb2c6a7cdba8f8a8aa95b8d6b9c2db51R45.

--- a/tests/functional.mk
+++ b/tests/functional.mk
@@ -42,7 +42,7 @@ $(FTST_IMAGE): FTST_IMAGE_ROOTFSDIR := $(FTST_IMAGE_ROOTFSDIR)
 $(FTST_IMAGE): $(FTST_IMAGE_MANIFEST) $(FTST_ACI_INSPECT) $(FTST_ACI_ECHO_SERVER) | $(FTST_IMAGE_TEST_DIRS)
 	echo -n dir1 >$(FTST_IMAGE_ROOTFSDIR)/dir1/file
 	echo -n dir2 >$(FTST_IMAGE_ROOTFSDIR)/dir2/file
-	ln -s /inspect $(FTST_IMAGE_ROOTFSDIR)/inspect-link
+	ln -sf /inspect $(FTST_IMAGE_ROOTFSDIR)/inspect-link
 	"$(ACTOOL)" build --overwrite "$(FTST_IMAGE_DIR)" "$@"
 
 BGB_BINARY := $(FTST_INSPECT_BINARY)

